code42.json

```diff
@@ -629,6 +629,242 @@
+            "undo": "unblock user",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "username": {
+                    "description": "Username to block",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "value_list": [],
+                    "default": "",
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.username",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "Username",
+                    "column_order": 0
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "deactivate user",
+            "identifier": "deactivate_user",
+            "description": "Deactivates user's Code42 account",
+            "verbose": "",
+            "undo": "reactivate user",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "username": {
+                    "description": "Username to deactivate",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "value_list": [],
+                    "default": "",
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.username",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "Username",
+                    "column_order": 0
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "reactivate user",
+            "identifier": "reactivate_user",
+            "description": "Reactivates a deactivated user's Code42 account",
+            "verbose": "",
+            "undo": "deactivate user",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "username": {
+                    "description": "Username to reactivate",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "value_list": [],
+                    "default": "",
+                    "order": 0
+                }
+            },
+            "output": [
```

Comment:
Probably should have some sort of summary mentioned in the output, even if it's just the success message.
I believe the success message counts as the summary if you include this:

```
, 
{
  "data_path": "action_result.summary",
  "data_type": "string"
}

Same with `unblock` and any others
```